client/src/components/BookSitter/BookSitter.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Typography,
+  Grid,
+  CircularProgress,
+  Paper,
+  Card,
+  CardMedia,
+  CardContent,
+  Box,
+  TextField,
+  Button,
+} from '@material-ui/core';
+import { Rating } from '@material-ui/lab';
+import { useAuth } from '../../context/useAuthContext';
+import house from '../../Images/house.jpg';
+import useStyles from './useStyles';
+
+const BookSitter = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+
+  return (
+    <Card className={classes.mainCard} elevation={5}>
+      <Paper elevation={3} className={classes.paper}>
```

Comment:
I think we can exclude the Paper component over here? As cards have elevation and box-shadows by default.

Reply:
Yes, I removed it without any impact - thank you!